### ISSUE TITLE
Navbar - Beginner / Moderate

### ISSUE THREAD

**Issue by markdreed512** [BaltimoreCode-Coffee/BaltimoreCode-Coffee.github.io#3]
Subtasks:
- Add C&C logo
- update link text (confirm what these links should be)
- Add dropdown functionality to any links with sub-nav
- Add styling to match design file
- Mobile view (this could be added as a separate issue once the main nav is finished. Are we able to get a design mockup for the mobile nav?)

I would tag this as a beginner / moderate skill level. The most challenging aspects would be the dropdown sub-nav and the mobile view (which can be added as a separate task anyway)

**Comment by markdreed512** [comment]
Task created in Jira.
